homeassistant/components/isy994/fan.py

```diff
@@ -98,8 +105,3 @@ def turn_on(self, speed: str = None, **kwargs) -> None:
         """Send the turn off command to ISY994 fan program."""
         if not self._actions.runElse():
             _LOGGER.error("Unable to turn on the fan")
-
-    @property
```

Comment:
Looks like this isn't provided by `ISYProgramEntity`. Does this need to be kept in this PR?

Reply:
It's provided by [FanEntity](https://github.com/home-assistant/core/blob/34e35f6aa3cc7d98f14223b80d6346cc0d9c36dd/homeassistant/components/fan/__init__.py#L189).

Previously the program class inherited from the ISYFanDevice, so ISYFanDevice overloaded the base FanEntity and added speed support, and this was here to overload it back to default values.